docs/core/diagnostics/metrics-collection.md

```diff
@@ -0,0 +1,404 @@
+---
+title: Collect metrics - .NET
+description: Tutorial to collect metrics in .NET applications
+ms.topic: tutorial
+ms.date: 10/27you/2021
```

Comment:
```suggestion
ms.date: 10/27/2021
```

Reply:
that is interesting, not sure how that got there : )